docs/fieldnorm/life_cycle.md

```diff
@@ -1,7 +1,56 @@
+Smaller scale software development and maintenance cycles that might not affect other areas of a genetic lab should maintain a similar strategy on a smaller scale and less laborious, and might use tooling that is more effective for software development, such as code based documentation, an online documentation system that are more suitable for software development planning (like confluence, or wikipages), an issue management system (like issue manager or jira) or combinations thereof.
+Guidelines should cover both strategies, large scale and small scale.
 
-Donec id nibh aliquam, euismod velit id, aliquam ante. Vestibulum finibus interdum tellus quis efficitur. Quisque convallis nec nisl ut sollicitudin. Sed fringilla tellus et massa viverra interdum. Curabitur egestas diam at dui consectetur, et dapibus dui dignissim. Maecenas eget pulvinar enim. Etiam sodales id dui id venenatis. Duis id pellentesque tellus. Quisque enim justo, tristique id volutpat in, euismod ac tellus. Etiam faucibus velit sed purus commodo, eu vehicula velit egestas. Vivamus faucibus est et tortor imperdiet convallis. Vestibulum nulla enim, blandit a purus eget, cursus pellentesque lectus. Nulla finibus mi euismod eros lobortis, vitae eleifend erat rutrum.
\ No newline at end of file
+2.      Requirements
+This process involves gathering, documenting, and analysing the software requirements. It includes capturing both functional and non-functional requirements that the software must fulfil to meet the intended use and comply with regulatory standards. One strategy for capturing this, which is also well suited for genetic laboratories is to use user stories, as stakeholders within a genetic laboratory have very specific needs but are often not able to translate those in well defined technical requirements.
+
+Guidelines should support the following processes to gather, convert, track and update the requirements during a project:  
+
+  a.       Collaborate closely with end users of the software when defining requirements.
+
+  b.       Convert end users' requirements into small individual development tasks.
+
+  c.       Utilize a task management system integrated with your Git development system to manage tasks.
+
+  d.       Maintain communication with end users throughout the development life cycle in order to prevent unexpected adjustments to requirements in later stages of development.
+
+A widely adopted software development strategy involves dividing a development project over several periods. It is advised to use an approach that enables to release partial deliverables to users each period, which enables user feedback upon each deliverable during the entire lifetime of the project, and provides ample opportunities for clarification of requirements or adapt their priorities.   
```

Comment:
do we want all bioinformaticians to work with 'user stories'? or could this be an option? 

General note: I would like to avoid "genetic laboratories", as this fieldnorm might be used for pathology or other laboratories as well

Reply:
general note: if this becomes a fieldnorm; everyone needs to adjust. 
So I would avoid trying to make it perfect, and leave room for different ways of working...  
We need to make clear where we need documentation.

Reply:
> You can also refer to the 'requirements and configurationmanagement'-section

**I have added a referral to the requirements 🙂**

> do we want all bioinformaticians to work with 'user stories'? or could this be an option?
**I have made it an example, so optional**

> General note: I would like to avoid "genetic laboratories", as this fieldnorm might be used for pathology or other laboratories as well
**True, I have changed it to either "laboratories" or "medical laboratories" **